testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/authorization/DefaultAuthzConfigAdapterTest.java

```diff
@@ -103,7 +104,7 @@ public void testDefaultAuthzConfig() throws Exception {
     }
 
     private void login() throws MalformedURLException {
-        this.driver.navigate().to(getResourceServerUrl());
+        this.driver.navigate().to(getResourceServerUrl() + "/");
```

Comment:
Why this is needed?

Reply:
Because in undertow, probably with jsp (jastow), it doesn't work without / at the end of url.